### DISCUSSION TITLE
colored lines on picture

### DISCUSSION THREAD

**Discussion by Zeench** [jomjol/AI-on-the-edge-device#3090] [Q&A]
Hi guys,

sorry my english is not the best, but i think the uploaded picture will tell you what my problem is.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/74009385/b0cc55d7-23ec-43cd-ac96-575af0fe3148)

These colored lines pop up on like 90% of the pictures taken.

Does someone have any idea how to solve this issue and what causes it?

Thank you in advance

edit: on the live stream are also lines, but they appear and disappear randomly.

**Comment by SybexX** [comment]
it could be a contact problem.
Check your camera connector to see if it is seated correctly.
